tensor2tensor/models/research/universal_transformer_util.py

```diff
@@ -1226,460 +1209,6 @@ def should_continue(u0, u1, halting_probability, u2, n_updates, u3):
 
   return new_state, (ponder_times, remainders)
 
-
```

Comment:
The code below are interesting and used functions, please do not remove it unless there's an equivalent.